test/core/experimental/vectors/test_vector.py

```diff
@@ -6,10 +6,10 @@
 from symplyphysics.core.experimental.vectors import (
     VectorSymbol,
     ZERO,
-    norm,
+    VectorNorm as norm,
```

Comment:
I suggest to instead make helper functions, i.e.
```
def norm(vector: VectorExpr, **kwargs: Any):
    return VectorNorm(vector, kwargs)
```

Now it may be confusing to see an object created when you expected to call a function `norm()`